src/components/LeftSidebar/LeftSidebar.astro

```diff
@@ -76,4 +78,10 @@ for (const section of sidebarSections) {
 			display: none;
 		}
 	}
+	
+	@media (min-width: 72em) {
+		:global(.left-sidebar) {
```

Comment:
Would it make sense to set this style in `CommunityMenu.astro` as that’s where the `left-sidebar` class is set? I think that might also avoid the need for `:global`.

Reply:
(we could also consider renaming the class and component prop `left-sidebar` → `hide-on-large-screens` or `lg:display-none` or something)

Reply:
Yes, it sure makes a lot more sense, but... It just doesn't work for some reason!? (or I am too dumb to understand). My first attempt was to make this media query part of the `CommunityMenu` component, but the styles weren't being applied. Then, I just tried to make this part of `LeftSidebar` and it worked, for some other mysterious reason.

Reply:
OK, then leave it for now. Once we have the new position of the community section sorted I’ll pull this branch and see if I can figure out what’s going on 🙌